doc/getting_started/installation_win.rst

```diff
@@ -140,15 +154,18 @@ environment for Windows. Follow the steps below to set it up:
     .. code-block:: console
 
        $ cd $ZEPHYR_BASE/samples/hello_world
-       $ make BOARD=arduino_101
+       $ cmake -Bbuild -H. -DPREBUILT_HOST_TOOLS=$ZEPHYR_BASE/build  -DBOARD=arduino_101
```

Comment:
Perhaps we should recommend users to

export PREBUILT_HOST_TOOLS=$ZEPHYR_BASE/build

so they don't think they have to have this on the CLI

Reply:
I'd rather not because we ask them to execute zephyr_env.sh, which allows for mutliple trees